appinfo/database.xml

```diff
@@ -124,6 +124,12 @@
 				<unsigned>true</unsigned>
 				<length>4</length>
 			</field>
+			<field>
+				<name>template_destination</name>
+				<type>integer</type>
+				<notnull>false</notnull>
+				<length>4</length>
+			</field>
```

Comment:
Do we still use database xml?
I thought we killed the feature in profit or migrations?

Reply:
Yes we do for now. As 13 is still supported